extension/js/inspector/app/modules/Data.js

```diff
@@ -67,6 +70,7 @@ define([
 
       var modelData = event.data;
       this.modelCollection.add(modelData);
+      this.classCollection.add(modelData);
```

Comment:
@jasonLaster Not sure I understand item 2.  We are not currently getting dupes of the class models because we are adding the model and the model `id_attr` is the class ID, and BB only add to the collection if id is unique.  

I could be misunderstanding what you're saying.